incubator/cloudbeaver/README.md

```diff
@@ -0,0 +1,156 @@
+
+
+* **`files/cloudbeaver/cloudbeaver.auto.conf`**
+
+   * Contains some automatic initial server configuration for Cloudbeaver.  Some of these entries are configured in the values file, under `cloudbeaver.configs`.   The presence of this file causes Cloudbeaver to skip the initial configuration on first login.  See this page for more information: https://github.com/dbeaver/cloudbeaver/wiki/Server-configuration#automatic-server-configuration
+
+
+* **`files/cloudbeaver/initial-data.conf`**
+
+   * Contains initial configuration for the admin user and roles.  By default, anonymous access is possible, and users can be created by the admin user after first start up.  The Cloudbeaver administrator user can also add users after deployment.  These entries are configured in the values file, under `cloudbeaver.configs.initialData`.  See this page for more information: https://github.com/dbeaver/cloudbeaver/wiki/Users
+
+
+* **`files/cloudbeaver/initial-data-sources.conf`**
+
+   * Contains a default template datasource for each of the chosen NuoDB deployments, which is created using the values provided for the existing or expected NuoDB deployments.  Users will also be able to add their own additional data sources, for example if more databases are deployed after the initial deployment of Cloudbeaver.  These entries are configured in the values file, under `cloudbeaver.configs.initialDataSources`.  See this page for more information: https://github.com/dbeaver/cloudbeaver/wiki/Configuring-server-datasources
+
+**NOTE:** The above 4 configurations can be optionally disabled using `.Values.cloudbeaver.configs.override`.  Setting to false will use all Cloudbeaver default configuration files.
+
+
+* **`files/io.cloudbeaver.resources.drivers.base/plugin.xml`**
+
+   * This file has this 3 additional lines for NuoDB:
+
+     - `<resource  name="drivers/nuodb"/>`
+     - `<bundle  id="drivers.nuodb"  label="NuoDB drivers"/>`
+     - `<driver  id="generic:nuodb"/>`
+
+* **`files/org.jkiss.dbeaver.ext.generic/plugin.xml`**
+
+    * This file has 1 additional section for NuoDB:
+
+       ```xml
+       <driver
+           id="nuodb"
+           label="NuoDB"
+           icon="icons/nuodb_icon.png"
+           iconBig="icons/nuodb_icon_big.png"
+            class="com.nuodb.jdbc.Driver"
+           sampleURL="jdbc:com.nuodb://{host}[:{port}]/[{database}]"
+           defaultPort="48004"
+           description="NuoDB"
+           webURL="http://www.nuodb.com/"
+           categories="sql,cloud">
+         <file  type="jar"  path="drivers/nuodb"/>
+       </driver>
+       ```
+
+### Deployment
+
+Install the chart with helm, for example:
+
+```sh
+helm upgrade cloudbeaver ~/git/nuodb-helm-charts/incubator/cloudbeaver/ --namespace cloudbeaver --install --create-namespace
```

Comment:
To align with the README.md files for the other charts, we may want to omit the `~/git/nuodb-helm-charts/` prefix.

Reply:
Done!